### PR TITLE
systemd: resolve issue #5408 by specifying libgpg-error path

### PR DESCRIPTION
**Pull request by Char-Aznable**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Please bump the revision to 2, and remove the bottle block so that a new bottle gets build. Thanks!

**Comment by Char-Aznable** [comment]
Is there any update on this? I hit this bug again on another machine...

File: Formula/systemd.rb [hunk 1]
```diff
@@ -4,13 +4,9 @@ class Systemd < Formula
   url "https://github.com/systemd/systemd/archive/v234.tar.gz"
   sha256 "da3e69d10aa1c983d33833372ad4929037b411ac421fb085c8cee79ae1d80b6a"
   head "https://github.com/systemd/systemd.git"
-  revision 1
+  revision 2
   # tag "linuxbrew"
 
-  bottle do
-    sha256 "af6857ce54506942d7944518442b40b767a0ce6dbfba7f6c2dbc20f22b8f7928" => :x86_64_linux
-  end
```
Comment (iMichka) [inline Formula/systemd.rb]: For this to work, just remove the sha256, else the bottle will not be uploaded by our CI:
```
bottle do
end
```

**Comment by iMichka** [comment]
Woups sorry forgot about that because we had other emergencies with brew 1.5.0 and 1.5.1 that needed urgent fixes.

The bottle did not upload because CI was not happy. We need to leave the bottle block. Thanks, and sorry again.

**Comment by iMichka** [comment]
Merged! I squashed all your commits into a single commit. Thanks for your contribution!

I will build the bottle separately, this PR is a little bit older and bottle upload does not work in that case (a rebase would have been needed, but I preferred to merge this and build the bottle here: #5866)

**Pull request closed without merging**